Directory.Build.props

```diff
@@ -37,7 +37,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Nerdbank.GitVersioning" PrivateAssets="none" />
+    <PackageReference Include="Nerdbank.GitVersioning" PrivateAssets="all" />
```

Comment:
Does this cause the `Nerdbank.GitVersioning` assembly to be copied to the project output directory?

Reply:
No, I do not see it being copied. But it avoids the dependency entries in the *.nuspec files.
Using `PrivateAssets="all"` is recommend by the [`Nerdbank.GitVersioning` docs](https://github.com/dotnet/Nerdbank.GitVersioning/blob/main/doc/msbuild.md).